airflow/plugins_manager.py

```diff
@@ -122,3 +135,15 @@ def make_module(name, objects):
     admin_views.extend(p.admin_views)
     flask_blueprints.extend(p.flask_blueprints)
     menu_links.extend(p.menu_links)
+
+# Merge base extra links with operators' own extra links
+for operators_module in operators_modules:
```

Comment:
I don't love this if I'm honest. But it's probably okay.

Reply:
I don't like it either. I put the `extra_links` to be under `AirflowPlugin` so that users can configure it in the same place as usual, which is nice to users I guess. However, that means both `extra_links` and customized operators are loaded at the same time and we need to do this.